ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -216,15 +216,15 @@ def split(tensor, split_size_or_sections, dim=0):
 
 
 def _get_indices_or_sections(indices_or_sections, indices, sections):
-    if not indices_or_sections:
-        if isinstance(indices_or_sections, (list, tuple)):
-            pass
-        elif indices and not sections:
+    if not isinstance(indices_or_sections, (list, tuple, int)):

```

Comment:
`if not ivy.exists(indices_or_sections):` is probably even better, I updated the previous comment but was too late it seems, sorry for that, we can merge after this little tweak:)